cudamapper/src/main.cu

```diff
@@ -134,7 +134,7 @@ int main(int argc, char* argv[])
         std::cerr << "NOTE - Since query and target files are same, activating all_to_all mode. Query index size used for both files." << std::endl;
     }
 
-    std::shared_ptr<claragenomics::io::FastaParser> query_parser = claragenomics::io::create_kseq_fasta_parser(query_filepath);
+    std::shared_ptr<claragenomics::io::FastaParser> query_parser = claragenomics::io::create_kseq_fasta_parser(query_filepath, k + w);
```

Comment:
Shouldn't this be `k + w - 1`?
For `k = 5, w = 1` 5 basepairs are needed, for `k = 5, w = 2` 6 and so on